webapp/templates/proxy-create.xhtml

```diff
@@ -0,0 +1,303 @@
+            <input type="hidden" property="calli:alternate" content="?alternate" value="{?alternate}"/><!-- special case for viewing when no GET rule -->
+            <input type="hidden" property="calli:copy" content="?copy" value="{?copy}"/>
+            <input type="hidden" property="calli:post" content="?post" value="{?post}"/>
+            <input type="hidden" property="calli:put" content="?put" value="{?put}"/>
+            <input type="hidden" property="calli:patch" content="?patch" value="{?patch}"/>
+            <input type="hidden" property="calli:delete" content="?delete" value="{?delete}"/>            
+        </div>
+        <input id="cacheProperty" type="hidden" property="calli:cacheControl" onchange="calli.updateProperty(event, 'calli:cacheControl')"/>
+                
+        <div class="row">
+            <div id="rules" class="col-sm-12">
+                <label>Handlers</label>
+
+                <div class="panel panel-default rule" data-method="GET">
+                    <div class="panel-heading">
+                        <h3 class="panel-title">GET Handler
+                            <span class="pull-right">
+                                <a href="#getHandler" data-toggle="collapse" class="glyphicon glyphicon-minus"></a>
+                            </span>
+                        </h3>
+                    </div>
+                    <div class="panel-body collapse in" id="getHandler">
+                        <fieldset class="row">
+                            <div class="form-group col-sm-2">
+                                <label for="getActionType">Action</label>
+                                <div>
+                                    <select id="getActionType" class="form-control actionType">
+                                        <option value="calli:copy">Copy (200)</option>
+                                        <option value="calli:post">POST</option>
+                                        <option value="calli:put">PUT</option>
+                                        <option value="calli:patch">PATCH</option>
+                                        <option value="calli:delete">DELETE</option>
+                                    </select>
+                                </div>
+                            </div>
+                            <div class="form-group col-sm-7">
+                                <label for="getPattern">URL Pattern</label>
+                                <div>
+                                    <input type="text" class="form-control requestPattern" placeholder="Pattern for variables"/>
+                                </div>
+                            </div>
+                            <div class="form-group col-sm-3">
+                                <label for="cache">Cache control</label>
+                                <div>
+                                    <select id="cache" class="form-control">
```

Comment:
Instead of using a hidden property, add onchange handler directly to select element.
```
<select id="cache" class="form-control" onchange="calli.updateProperty(event,'calli:cacheControl')">
<option property="calli:cacheControl" selected="selected" content="?cache" value="{?cache}">{?cache}</option>
<!-- above option is used in edit form to select existing value, might need js to change option text to something more friendly -->
<option value=""></option>
<option value="no-store">Don't cache</option>
<option....
```